docs/DeveloperGuide.md

```diff
@@ -194,14 +194,42 @@ This section describes some noteworthy details on how certain features are imple
 ### 3.1 Exporting of information
 
 #### 3.1.1 Implementation
-Currently, only email address and phone number can be exported. The key idea is that we will iterate through the current list and access the relevant information fields.
-We will then write the information into a .txt file located at `/data/hall.txt` each separated by a new line. In the event that the current list is empty, an empty .txt file will be produced.
+The export feature is facilitated by `FileWriter` from Java's IO library.
+Currently, only email address and phone number can be exported. 
+
+The key idea is that we will iterate through the current list and access the relevant information fields.
+This operation depends on the size of the current person list and will be relatively fast.
+We will then write the information into a .txt file located at `/data/hall.txt` each separated by a new line. 
+
+Given below is a step-by-step usage scenario of how the `export` feature works:
+
+Step 1. The user launches the application and inputs `export email` into the input box.
+
+Step 2. The `LogicManager#execute()` is then called and the input is parsed through `AddressBookParser#parseCommand()`, returning an `ExportCommand`.
+
+Step 3. The `export` command then calls `ExportCommand#execute()`, and calls `Model#getAddressBook()` follwed by `ReadOnlyAddressBook#getPersonList()` to get the current list of persons.
+
+Step 4. The person list is then passed to `ExportCommand#handlEmail()` which iterates through the list and calls `Person#getEmail()` to access the `Email` and writes to the file `hally.txt`
```

Comment:
Is the format consistent with the numbering style in the logic API?

Reply:
Ok done